### PR TITLE
Add hypha's covid response

### PR DESCRIPTION
**Pull request by dcwalk**

### CODE REVIEW

File: SUMMARY.md [hunk 1]
```diff
@@ -12,6 +12,7 @@
   - [Roles](roles.md)
   - [Working Open](working-open.md)
   - [Finance 🚧](finance.md)
+  - [Novel Coronavirus (2019) 🦠](coronavirus.md)
```
Comment (benhylau) [inline SUMMARY.md:15]: What's the emoji, I cannot see it.

File: coronavirus.md [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+
+Like others in the Great Lakes region and Toronto, we’ve been closely monitoring the Novel Coronavirus 2019 situation. With the pace the situation has evolved since the World Health Organization [declared COVID-19 a pandemic on March 11](https://www.who.int/dg/speeches/detail/who-director-general-s-opening-remarks-at-the-media-briefing-on-covid-19---11-march-2020), we can assume that things will continue to change rapidly in the weeks and months ahead.
+
+In order to stay safe as workers and community members during this we've drafted this document, which contains our pandemic-specific policies and guidance, resources, and information on business and personal supports for responding to COVID-19.
+
+- [Working in the Time of COVID-19](#working-in-the-time-of-covid-19) 
+- [Communication and Education](#communication-and-education) 
+- [Health and Safety Considerations](#health-and-safety-considerations)
+  - [Social Distancing and Self Isolation](#social-distancing-and-self-isolation) 
+  - [Personal Protective Equipment (PPE)](#personal-protective-equipment-ppe)
```
Comment (benhylau) [inline coronavirus.md:11]:
```
- ./_book/coronavirus.html
  *  linking to internal hash #Personal-Protective-Equipment-PPE that does not exist (line 0)

     <a href="#Personal-Protective-Equipment-PPE">Personal Protective Equipment (PPE)</a>
htmlproofer 3.15.2 | Error:  HTML-Proofer found 1 failure!
```

File: coronavirus.md [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+## Working in the Time of COVID-19
+
+As a remote-first organization with many member-owners additionally working outside Hypha we occupy a slightly unique position. On the one hand our own work can transition to a work from home (WFH) strategy, but the continuity of Hypha work may have additional impacts we should be mindful of.
+
+After a process of research, consultation, and review of our requirements as an employer, Hypha has adopted the following policy to protect ourselves as member-owners and to protect the communities we serve:
+
+- Member-owner's work should continue from home/safe environments
+    - Client work and internal meetings will continue as virtual meetings
+    - Work-related travel or professional gatherings/onsite meetings will be moved to virtual hosting or be rescheduled to a time when public health officials have deemed it safe to do so
+    - Due to the nature of our work (virtual office, remote) we **are not providing** [Personal Protective Equipment (PPE)](#Personal-Protective-Equipment-PPE) at this time
```
Comment (benhylau) [inline coronavirus.md:30]:
```
- ./_book/coronavirus.html
  *  linking to internal hash #Personal-Protective-Equipment-PPE that does not exist (line 0)

     <a href="#Personal-Protective-Equipment-PPE">Personal Protective Equipment (PPE)</a>
htmlproofer 3.15.2 | Error:  HTML-Proofer found 1 failure!
```

File: coronavirus.md (hunk 2 above)
Comment (benhylau) [reply coronavirus.md:11]: Not sure if this one is affected too.

Comment (dcwalk) [reply coronavirus.md:11]: this one works I think! been testing locally

File: SUMMARY.md (hunk 1 above)
Comment (dcwalk) [reply SUMMARY.md:15]: Screenshot_2020-04-15_09-25-26

File: coronavirus.md (hunk 3 above)
Comment (dcwalk) [inline coronavirus.md:30]:
```suggestion
    - Due to the nature of our work (virtual office, remote) we **are not providing** [Personal Protective Equipment (PPE)](#personal-protective-equipment-ppe) at this time
```

File: SUMMARY.md (hunk 1 above)
Comment (dcwalk) [reply SUMMARY.md:15]: Should I cut? It's around in most places! 
> "Microbe was approved as part of Unicode 11.0 in 2018 and added to Emoji 11.0 in 2018. "

File: coronavirus.md [hunk 4]
```diff
@@ -0,0 +1,158 @@
+# Responding to the Novel Coronavirus (2019) Pandemic 
+
+Like others in the Great Lakes region and Toronto, we’ve been closely monitoring the Novel Coronavirus 2019 situation. With the pace the situation has evolved since the World Health Organization [declared COVID-19 a pandemic on March 11](https://www.who.int/dg/speeches/detail/who-director-general-s-opening-remarks-at-the-media-briefing-on-covid-19---11-march-2020), we can assume that things will continue to change rapidly in the weeks and months ahead.
+
+In order to stay safe as workers and community members during this we've drafted this document, which contains our pandemic-specific policies and guidance, resources, and information on business and personal supports for responding to COVID-19.
```
Comment (dcwalk) [inline coronavirus.md:5]:
```suggestion
In order to stay safe as workers and community members during this we've drafted this document, which contains our pandemic-specific policies and guidance, resources, and information on business and personal supports for responding to COVID-19. It was last updated on **April 15, 2020**.
```

**Comment by dcwalk** [comment]
Okay I'm going to merge this now, and we can decide if it needs to be revised later!

File: SUMMARY.md (hunk 1 above)
Comment (benhylau) [reply SUMMARY.md:15]: Hah!

**Pull request merged**